Conrollers/getConroller.js

```diff
@@ -20,24 +21,24 @@ function getapi(req, res, next) {
   res.status(200).send({ msg: "No content found" });
 }
 function getArticles(req, res, next) {
-  getArticlesMdl(req)
+  const { sortBy, order } = req.query;
```

Comment:
the ticket requires a topic option too, no?

Reply:
because you've not tested for it, this topic filter feature is presently broken in your app